validation.css

```diff
@@ -0,0 +1,29 @@
+
+.is-form-valid {
+  border: 1px solid limegreen;
+  border-left: 8px solid limegreen;
+  box-shadow: 0 0 5px limegreen;
+}
+
+.is-form-invalid {
+  border: 1px solid orangered;
+  border-left: 8px solid orangered;
+  box-shadow: 0 0 5px orangered;
+}
+
+.text-success {
+  padding-left: 10px;
+  color: #fff;
+  font-size: 14px;
+  position: relative;
+  bottom: 22px;
+}
+
+.text-danger {
+  padding-left: 10px;
+  color: darkred;
+  font-size: 15px;
+  font-weight: 200;
+  position: relative;
+  bottom: 22px;
+}
```

Comment:
- [ ] Please make sure that your contact form stick to the design guidelines (e.g., font, colors, margins) of [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) as required for this project you can see a bit differences between your design and the chosen Figma :+1: 

#### Your design
![Screenshot from 2022-11-17 21-36-17](https://user-images.githubusercontent.com/6615394/202564845-bb4b3fce-29aa-44ad-b6fe-03025d2d7ed7.png)

#### Figma design
<img width="310" alt="contact_form" src="https://user-images.githubusercontent.com/6615394/202565020-695a1f9d-9785-4510-bdc3-ba57504e02ac.png">